tests/conftest.py

```diff
@@ -686,6 +687,14 @@ def _fetch_json(url, params=None):
     monkeypatch.setattr(treeherder.etl.bugzilla, 'fetch_json', _fetch_json)
 
 
+@pytest.fixture
+def mock_deviance(monkeypatch):
```

Comment:
Could you add a comment here that specifies that this should only be used when dealing with a time-series of constant values?

We should also file an issue to add a test here that uses static, but non-constant data like we do in moz-measure-noise: https://github.com/mozilla/measure-noise/blob/dev/tests/test_basic.py#L20-L36

Reply:
Jira ticket: https://mozilla-hub.atlassian.net/browse/FXP-2626